integration/maptest/hammer.go

```diff
@@ -0,0 +1,678 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
This looks like it's setting up various `hammer` objects.  Perhaps this should live in a `hammer` package?  Perhaps `testonly/hammer`? As it is `maptest` holds unittests exclusively.

Reply:
I'm hoping to use some of the `maptest` utility functions for verification (e.g. `verifyGetMapLeavesResponse` looks potentially useful).  But any such utilities should be promoted to be publicly visible and then the hammer code can live on its own.

Reply:
Hmm, good point. I think this is a hint that we need something of the form of `client/mapverifier` which all clients can use.